src/python/pants/backend/python/target_types.py

```diff
@@ -744,14 +744,10 @@ class PythonTestsOverrideField(OverridesField):
 
 class PythonTestsGeneratorTarget(TargetFilesGenerator):
     alias = "python_tests"
-    core_fields = (
-        PythonTestsGeneratingSourcesField,
-        InterpreterConstraintsField,
-        PythonTestsOverrideField,
-    )
+    core_fields = (PythonTestsGeneratingSourcesField, PythonTestsOverrideField)
     generated_target_cls = PythonTestTarget
-    copied_fields = (InterpreterConstraintsField,)
-    moved_fields = _PYTHON_TEST_MOVED_FIELDS
+    copied_fields = ()
+    moved_fields = (*_PYTHON_TEST_MOVED_FIELDS, InterpreterConstraintsField)
```

Comment:
Should be able to put `InterpreterConstraintsField` in `_PYTHON_TEST_MOVED_FIELDS` instead.